assets/batch_score/components/driver/src/batch_score/metadata.json

```diff
@@ -1,5 +1,5 @@
 {
-  "component_version": "1.0.4",
+  "component_version": "1.0.3",
```

Comment:
I assume this will be bumped to 1.1.0.

Reply:
Please bump the version in batch_score_llm/spec.yaml too.

Reply:
done, thanks